index.html

```diff
@@ -465,14 +465,9 @@ <h2>cnf</h2>
       If <code>cnf</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>
       or the <a data-cite="RFC7519#section-4.1.1">JWT Claims </a>,
       a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
```

Comment:
```suggestion
      a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> MAY use this parameter
```

Reply:
@selfissued ^ can you comment on this suggestion?